### DISCUSSION TITLE
Deploy FastAPI on VPS

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#899] [Q&A] [answered]
Saya punya aplikasi FastAPI yang berjalan di lokal. Saya ingin agar FastAPI saya berjalan di VPS ubuntu. Saya juga sudah menyiapkan domain. Langkah-langkah apa yang harus saya lakukan agar FastAPI saya bisa berjalan di VPS ?

Mohon bantuannya, terima kasih.

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear @yohanespm77 
Terima kasih telah bertanya di Github Community Algoritma.

Terkait pertanyaan Anda, berikut kami lampirkan resource yang sekiranya dapat membantu:
1. Medium: [Deploy FastAPI on VPS](https://lemoncode21.medium.com/deploy-fastapi-on-vps-ff482a3c52ce).
2. YouTube [Deploy Fastapi on VPS](https://www.youtube.com/watch?v=vlMJhxI0qkA)

Semoga jawaban ini dapat membantu.

Best regards,
Team Algoritma

**Comment by adizaen** [comment]
Dear @yohanespm77 

Untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.

Best regards,
**Team Algoritma**
